### PR TITLE
Removing the double 'l' in "Cancelled" and "Cancelling"

### PR DESCRIPTION
**Pull request by AhmedLeithy**
As per #23088, the spelling decided on for the words: cancelled and cancelling is the (US) spelling of the word with one l.

Properties as well as enum values referring to the word have been changed. However the following instances of the words remain:

- The actual string value of the "DocumentStatus" Cancelled/Cancelling attribute. This is kept in order to ensure matching with the service response values.

- Very large docstrings in the generated "[DocumentTranslationRestClient.cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/translation/Azure.AI.Translation.Document/src/Generated/DocumentTranslationRestClient.cs)" contain the words cancelled and cancelling. [Autorest suggests redefining each properties that have such doc strings](https://github.com/chamons/autorest.csharp#changing-member-doc-comment) in a customized model, but this does not feel like an intuitive way to change the words, especially due to the size of the docstrings. Is there another way to change these and are they worth changing?

### CODE REVIEW

File: sdk/translation/Azure.AI.Translation.Document/src/StatusSummary.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.AI.Translation.Document.Models
+{
+    internal partial class StatusSummary
```
Comment (maririos) [inline sdk/translation/Azure.AI.Translation.Document/src/StatusSummary.cs:8]: this won't be needed as this type is internal and the suer will never know it exist :)

**Review by maririos — APPROVED** [review]

**Comment by maririos** [comment]
>The actual string value of the "DocumentStatus" Cancelled/Cancelling attribute. This is kept in order to ensure matching with the service response values.

This is ok as the way we expose it to the users is with one `l`.

> Very large docstrings in the generated "DocumentTranslationRestClient.cs" contain the words cancelled and cancelling. Autorest suggests redefining each properties that have such doc strings in a customized model, but this does not feel like an intuitive way to change the words, especially due to the size of the docstrings. Is there another way to change these and are they worth changing?

I don't believe we are exposing any of the "DocumentTranslationRestClient.cs" methods and therefore the docstrings so we should be ok to leave as is

File: sdk/translation/Azure.AI.Translation.Document/src/StatusSummary.cs (hunk 1 above)
Comment (AhmedLeithy) [reply sdk/translation/Azure.AI.Translation.Document/src/StatusSummary.cs:8]: Yea but I thought I might as well change it for consistency. Would you like me to revert this change?

Comment (maririos) [reply sdk/translation/Azure.AI.Translation.Document/src/StatusSummary.cs:8]: Yes. One less thing we will have to maintain in the future :)

**Review by maririos — APPROVED** [review]

**Pull request merged**